### PR TITLE
Back translation helper updates

### PR DESCRIPTION
**Pull request by bobeaton**
Fixed a few more issues on the same control (BackTranslationHelperCtrl) found while using it in Paratext.

### CODE REVIEW

File: src/BackTranslationHelper/BackTranslationHelperCtrl.cs [hunk 1]
```diff
@@ -1084,7 +1118,13 @@ private void ButtonPinToTop_Click(object sender, EventArgs e)
                                     ? global::BackTranslationHelper.Properties.Resources.pindown
                                     : global::BackTranslationHelper.Properties.Resources.pinup;
 
-            if ((Parent != null) && (Parent is Form parentForm))
+			// Now that I'm using a TableLayoutPanel in the Ptx plugin form, it may not be the parent.
+			//  So go up the parent chain until we find a Form
+			var parent = (Control)this;
```
Comment (jasonleenaylor) [inline src/BackTranslationHelper/BackTranslationHelperCtrl.cs:1123]:
```while(parent?.Parent != null && !parent is Form)
      parent = parent.Parent;
   if(parent is Form)
      parent.TopMost = newCheckState;
   ```
I suggest this alternative, fewer parenthesis, same behavior I think. 'Is' returns false on a null

File: src/BackTranslationHelper/BackTranslationHelperCtrl.cs [hunk 2]
```diff
@@ -1054,16 +1089,15 @@ private void AssignNewSubstitutionProjectMenuItem_Click(object sender, EventArgs
...
         }
 
-        private FindReplaceHelper TrySpellFixerProjectLogin(string findWhat)
+        private FindReplaceHelper TrySpellFixerProjectLogin()
         {
             try
             {
                 var aSF = FindReplaceHelper.GetFindReplaceHelper();
                 DirectableEncConverter.EncConverters.Reinitialize();
-                aSF.QueryForSpellingCorrectionIfTableEmpty(findWhat);
```
[the remark this answers is not in the document]
Comment (bobeaton) [reply src/BackTranslationHelper/BackTranslationHelperCtrl.cs:1066]: it doesn't want to anymore. (now that we use a dummy rule (\u001f => \u001f), it prevents the reason why we wanted to do that to begin with.

File: src/BackTranslationHelper/BackTranslationHelperCtrl.cs [hunk 3]
```diff
@@ -48,12 +48,19 @@ protected override void OnParentChanged(EventArgs e)
...
 
-            if (Properties.Settings.Default.PinnedToTop && (Parent != null) && (Parent is Form parentForm))
+            if (Properties.Settings.Default.PinnedToTop)
             {
+				var parent = (Control)this;
+				while (((parent = parent.Parent) != null) && !(parent is Form))
+					;
+
+				if ((parent != null) && (parent is Form parentForm))
+				{
```
[the remark this answers is not in the document]
Comment (bobeaton) [reply src/BackTranslationHelper/BackTranslationHelperCtrl.cs:58]: removed \t's again ... no idea how to turn that off effectively

File: src/BackTranslationHelper/BackTranslationHelperCtrl.cs (hunk 1 above)
Comment (bobeaton) [reply src/BackTranslationHelper/BackTranslationHelperCtrl.cs]: done

**Pull request merged**